Mathlib/Algebra/Star/SelfAdjoint.lean

```diff
@@ -182,6 +182,16 @@ variable [Monoid R] [StarMul R]
 theorem pow {x : R} (hx : IsSelfAdjoint x) (n : ℕ) : IsSelfAdjoint (x ^ n) := by
   simp only [isSelfAdjoint_iff, star_pow, hx.star_eq]
 
+@[grind =]
+lemma _root_.isSelfAdjoint_conjugate_iff_of_isUnit {a u : R} (hu : IsUnit u) :
+    IsSelfAdjoint (u * a * star u) ↔ IsSelfAdjoint a := by
+  simp [IsSelfAdjoint, mul_assoc, hu.mul_right_inj, hu.star.mul_left_inj]
+
+@[grind =]
+lemma _root_.isSelfAdjoint_conjugate'_iff_of_isUnit {a u : R} (hu : IsUnit u) :
```

Comment:
One last quibble: the prime looks strange in the middle of the name like this, it would be fine if `conjugate'` were its own concept, but it's just there to differentiate it with the previous one, it probably makes more sense to put it at the end.

Other than that, LGTM!

bors d+